test/topology_experimental_raft/test_fencing.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, Column, IntType, CounterType
+from test.pylib.util import unique_name
+from test.pylib.util import wait_for_cql_and_get_hosts
+from cassandra import WriteFailure
+
+import pytest
+import logging
+import time
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_fence_writes(request, manager: ManagerClient):
+    logger.info("Bootstrapping first node")
+    servers = [await manager.server_add()]
+
+    logger.info("Bootstrapping other nodes")
+    servers += [await manager.server_add(), await manager.server_add()]
+
+    logger.info(f'Creating new tables')
+    random_tables = RandomTables(request.node.name, manager, unique_name(), 3)
+    table1 = await random_tables.add_table(name='t1', pks=1, columns=[
+        Column("pk", IntType),
+        Column('int_c', IntType)
+    ])
+    table2 = await random_tables.add_table(name='t2', pks=1, columns=[
+        Column("pk", IntType),
+        Column('counter_c', CounterType)
+    ])
+    await manager.cql.run_async(f"USE {random_tables.keyspace}")
+
+    logger.info(f'Waiting for cql and hosts')
+    host2 = (await wait_for_cql_and_get_hosts(manager.cql, [servers[2]], time.time() + 60))[0]
+
+    version = (await manager.cql.run_async(
+        "select version from system.topology where key = 'topology'",
+        host=host2))[0].version
+    logger.info(f"version on host2 {version}")
+
+    await manager.cql.run_async(f"update system.topology set version={version - 1} where key = 'topology'",
```

Comment:
`wait_for_cql_and_get_hosts` waits for cql, cql is started after `storage_service::join_token_ring`, and `join_token_ring` waits for node to become `normal`. Here we get the version from the last, just bootstrapped node, it can become normal only on the last version.